dpcpp/components/cooperative_groups.dp.hpp

```diff
@@ -477,36 +480,38 @@ __dpct_inline__ grid_group this_grid(sycl::nd_item<3>& group)
 
 
 // Enable group can directly use group function
-__SYCL_INLINE_NAMESPACE(cl)
-{
-    namespace sycl {
-    namespace detail {
+#if GINKGO_DPCPP_MAJOR_VERSION < 6
+inline namespace cl {
+#endif
+namespace sycl {
+namespace detail {
 
 
-    template <unsigned Size>
-    struct is_sub_group<
-        ::gko::kernels::dpcpp::group::detail::thread_block_tile<Size>>
-        : std::true_type {};
+template <unsigned Size>
+struct is_sub_group<
+    ::gko::kernels::dpcpp::group::detail::thread_block_tile<Size>>
+    : std::true_type {};
```

Comment:
didn't notice this before - maybe we should check with Intel whether this could cause any issues on their side? Might break if their internals change?